deploy/frontend.yaml

```
@@ -7,31 +7,41 @@ objects:
   - apiVersion: cloud.redhat.com/v1alpha1
     kind: Frontend
     metadata:
-      name: digital-roadmap
+      name: roadmap
+
     spec:
-      API:
-        versions:
-          - v1
       envName: ${ENV_NAME}
       title: Starter App
       deploymentRepo: https://github.com/RedHatInsights/digital-roadmap
```

Could you also update this?
```suggestion
      title: Digital Roadmap
      deploymentRepo: https://github.com/RedHatInsights/digital-roadmap-frontend
```

---

Good catch. Thanks!